### PR TITLE
Add team event view #322

### PR DESCRIPTION
**Pull request by akaka0039**
## Content
#322 
#323 
・Fix sidebar design for event team
・Fix image and SVG files
## Image
Screenshot 2023-08-28 at 15 06 30
Screenshot 2023-08-28 at 15 06 05
Screenshot 2023-08-28 at 15 02 27
Screenshot 2023-08-28 at 15 02 56

## Related Issue
fixed

### CODE REVIEW

File: src/resources/js/Components/Pages/Event/EventDetail.jsx [hunk 1]
```diff
@@ -219,7 +220,7 @@ const EventDetail = ({
                                     </dt>
                                     <dd className="mt-1 text-lg text-gray-600">
                                         <div className="flex items-center">
-                                            <span className="truncate">
+                                            <span className="flex flex-row">
```
Comment (shin5446) [inline src/resources/js/Components/Pages/Event/EventDetail.jsx:223]: I made issue for here. You can fix this later on.
https://github.com/akaka0039/Global_Developer_site/issues/342

File: src/resources/js/Components/Pages/EventTeam/EventTeamList.jsx [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
 
+const EventTeamList = ({}) => {
     return (
         <>
+            <p>List page</p>
+            <div>
+                <Link href="/event_team/1" className="p-2 hover:text-gray-400">
+                    Event team 1
+                </Link>
+            </div>
```
Comment (shin5446) [inline src/resources/js/Components/Pages/EventTeam/EventTeamList.jsx:11]: @akaka0039 Could you make the List page for here in another PR?

**Review by shin5446 — COMMENTED** [review]
Thank you for writing the beautiful code. I want to confirm something then I put comments. Please check it out

**Comment by shin5446** [comment]
スクリーンショット 2023-08-30 午後4 07 59
this can be solved to change the path of the target image in MemberPop.jsx

File: src/resources/js/Components/Pages/EventTeam/EventTeamList.jsx (hunk 2 above)
Comment (akaka0039) [reply src/resources/js/Components/Pages/EventTeam/EventTeamList.jsx:11]: sure I'm gonna create another pr

**Review by shin5446 — APPROVED** [review]
LGTM
Perfect!

**Pull request merged**